src/Umbraco.Core/Services/FileService.cs

```diff
@@ -38,7 +40,7 @@ public FileService(
         IStylesheetRepository stylesheetRepository,
         IScriptRepository scriptRepository,
         IPartialViewRepository partialViewRepository,
-        IAuditRepository auditRepository,
+        IAuditService auditService,
```

Comment:
Looks like `shortStringHelper` and `globalSettings` aren't used in this class.  Can we remove these given we are already modifying the constructor?